### ISSUE TITLE
can't create issue without labels: nil is not a array

### ISSUE THREAD

**Issue by jwilk** [sociomantic/git-hub#142]
It's not possible to create an issue without any labels set:

```
$ git hub -v issue new
git command: ['git', 'config', 'hub.username'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'config', 'hub.oauthtoken'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'config', 'hub.upstream'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'config', 'hub.forkrepo'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'config', 'hub.forkremote'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'config', 'hub.pullbase'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'config', 'hub.urltype'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'config', 'hub.baseurl'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'config', '--bool', 'hub.forcerebase'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'config', '--bool', 'hub.triangular'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'config', 'hub.password'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'var', 'GIT_EDITOR'] {'stderr': -1, 'stdout': -1}
git command: ['git', 'rev-parse', '--git-dir'] {'stderr': -1, 'stdout': -1}
Request: POST https://api.github.com/repos/jwilk/testbed/issues
{"body": "...", "assignee": null, "milestone": null, "labels": null, "title": "test bug, please ignore"}
https://api.github.com/repos/jwilk/testbed/issues
Content-length: 104
Content-type: application/json
Authorization: <hidden>
Accept: application/vnd.github.v3+json
{"body": "...", "assignee": null, "milestone": null, "labels": null, "title": "test bug, please ignore"}
Error: GitHub error: Invalid request.

nil is not a array.
HTTP Error 422: Unprocessable Entity
https://api.github.com/repos/jwilk/testbed/issues
<httplib.HTTPMessage instance at 0xf6c6512c>
```

**Comment by mihails-strasuns-sociomantic** [comment]
Looks like GitHub API expects empty arrays instead of `null` for missing values here.

**Comment by Tecayehuatl** [comment]
Did you fix the Issue? I am trying to POST a Issue, I am using my token but I can't post the issue, it throws the next error: 

```
{
    "message": "Invalid request.\n\nFor 'links/0/schema', nil is not an object.",
    "documentation_url": "https://developer.github.com/v3/issues/#create-an-issue"
}
```

**Comment by leandro-lucarella-sociomantic** [comment]
It should have been, by https://github.com/sociomantic-tsunami/git-hub/commit/1137af4d6c61dc49261690d549122c878b98fc29. Can you add `-v` to confirm this is the same issue? Can you also make sure you have the latest version? The fix was first released in [v0.10.0](https://github.com/sociomantic-tsunami/git-hub/releases/tag/v0.10-rc1).
